web-registry/src/components/organisms/MediaForm.tsx

```diff
@@ -17,11 +24,14 @@ import {
   validate,
   getProjectPageBaseData,
   getCompactedPath,
+  getProjectShapeIri,
 } from '../../lib/rdf';
 import { useShaclGraphByUriQuery } from '../../generated/graphql';
 import getApiUri from '../../lib/apiUri';
 import { ProjectPageFooter } from '../molecules';
 import { useProjectEditContext } from '../../pages/ProjectEdit';
+import ControlledTextField from 'web-components/lib/components/inputs/ControlledTextField';
+import { FormHelperText } from '@mui/material';
```

Comment:
can you move this up? let's try to stick to this order for imports: external libs, web-components, internal